### PR TITLE
Abdullah Al Hommada - JS2 - Week2

### PR DESCRIPTION
**Pull request by aalhommada**

### CODE REVIEW

File: Week2/test/maartjes-work.test.js [hunk 1]
```diff
@@ -1,9 +1,69 @@
...
-describe('maartjes_work', () => {
-  test('earnings rounded to euro cents', () => {
-    const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
-    const result = earnings.toFixed(2);
-    expect(result).toBe('373.33');
-  });
-});
+'use strict';
+
+// const { maartjesTasks, maartjesHourlyRate, computeEarnings } = require(`../homework/maartjes-work`);
```
Comment (wilgert) [inline Week2/test/maartjes-work.test.js]: The idea of these tests is that they check your code. You should not comment them out. 
Also you don't need to copy/paste the data and functions from the `MAKEME.md`. This code already exist in `maartjes-work.js` and `map-filter.js`. You should implement your solutio there, then the tests can verify that your code does what it should do.

Comment (aalhommada) [reply Week2/test/maartjes-work.test.js]: I am not sure what should I do now , should I delete everything or just delete extra parts ?

Comment (wilgert) [reply Week2/test/maartjes-work.test.js]: You should change the `maartjes-work.test.js` and `map-filter.test.js` files back to how they were and put your solutions in `maartjes-work.js` and `map-filter.js`.

Comment (aalhommada) [reply Week2/test/maartjes-work.test.js]: Thanks Wilgert for your corrections , i will do that .

Comment (aalhommada) [reply Week2/test/maartjes-work.test.js]: Hi Wilgert , I rewrite everything , and the  ' npm run test-week2 ' work good.

Test Suites: 2 passed, 2 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        6.431s

Comment (wilgert) [reply Week2/test/maartjes-work.test.js]: That looks really good!

**Comment by fedefu** [comment]
good job @aalhommada

**Pull request closed without merging**